scipy/linalg/decomp_qr.py

```diff
@@ -37,8 +37,10 @@ def qr(a, overwrite_a=False, lwork=None, mode='full', pivoting=False,
     ----------
     a : (M, N) array_like
         Matrix to be decomposed
-    overwrite_a : bool, optional
-        Whether data in a is overwritten (may improve performance)
+    overwrite_a : bool, optional(default - False)
```

Comment:
The `False` default is also present/visible in the parameter list at the top, so I'm not sure if we need this addition. If we do add this, will at least need a space before the `(`.

Reply:
agreed, don't need this